doc/init.md

```diff
@@ -53,11 +53,12 @@ Paths
 
 All three configurations assume several paths that might need to be adjusted.
 
-Binary:              `/usr/bin/bitcoind`
-Configuration file:  `/etc/bitcoin/bitcoin.conf`
-Data directory:      `/var/lib/bitcoind`
-PID file:            `/var/run/bitcoind/bitcoind.pid` (OpenRC and Upstart) or `/run/bitcoind/bitcoind.pid` (systemd)
-Lock file:           `/var/lock/subsys/bitcoind` (CentOS)
+    Binary:              /usr/bin/bitcoind
```

Comment:
Binary:              /usr/bin/bitcoind
	Configuration file:  /etc/bitcoin/bitcoin.conf
	Data directory:      /var/lib/bitcoind
	PID file:            /var/run/bitcoind/bitcoind.pid (OpenRC and Upstart) or
                         /run/bitcoind/bitcoind.pip (systemd)
	Lock file:           /var/lock/subsys/bitcoind (CentOS)

Reply:
I recommend 4 spaces for indenting.

Reply:
Meh, he just want to merge this to promote himself. He literally only changed indenting. I think markdown files isn't important at all. Puff up as "fix". :laughing:

Reply:
I disagree - I wish more people reviewed documentation and submitted improvements.
This should be encouraged.

Reply:
You just disagree because you are an American terrorist at the @usnavy. The documentation is well written, there is nothing to fix. Adding a few spaces to get something merged in GitHub is absurd. You can run now to the Pentagon and cry about Zoltán Kondér.

What the hell a @usnavy has to do with Bitcoin anyway?

Reply:
This is formatting I'm looking to fix (below). It's indeed my first contribution albeit small. Looking forward to making more meaningful ones as I get more comfortable with the codebase.

![image](https://user-images.githubusercontent.com/7684574/106794252-77e75c00-6650-11eb-828c-fbe3df8be60f.png)

Reply:
I have linked to some suggested changes -> 

> Additional suggested improvements:
> 
> https://github.com/RandyMcMillan/bitcoin-core/blob/gunar-patch-1/doc/init.md

add those..

Reply:
@gunar :+1: I think there should be more changes than just adding spaces. At least write something..